Sources/ReplyService/FacebookGraph.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  FacebookGraph.swift
+//  SwiftBot
+//
+//  Created by Andrew Tokarev on 27/06/2017.
+//
+//
+
+import Foundation
+import PerfectCURL
+
+enum FacebookGraphRequestHttpMethod: Hashable, CustomStringConvertible {
+    
+    case get
+    case post
+    
+    public var description: String {
+        switch self {
+        case .get:      return "GET"
+        case .post:     return "POST"
+        }
+    }
+}
+
+struct FacebookGraphRequest {
+    
+    let path: String                // /me/messages
+    let params: [String: String]    // access_token:xxx
+    let body: Data                  // might be json
```

Comment:
may be we can make assumption that data will always be json, In general It can be image or something else, be we can handle it later